pkg/kubernetes/pods.go

```diff
@@ -156,7 +156,7 @@ func GetContainer(name string, spec *corev1.PodSpec) *corev1.Container {
 func GetPodMemoryLimitBytes(container, podName, namespace string, kube *Kubernetes) (uint64, error) {
 	execOut, err := kube.ExecWithOptions(ExecOptions{
 		Container: container,
-		Command:   []string{"cat", "/sys/fs/cgroup/memory/memory.limit_in_bytes"},
+		Command:   []string{"/bin/bash", "-c", "cat /sys/fs/cgroup/memory/memory.limit_in_bytes || cat /sys/fs/cgroup/memory.max"},
```

Comment:
Won't this result in the output containing something like:

```
cat: /sys/fs/cgroup/memory/memory.limit_in_bytes :No such file or directory
```

Followed by the value in the second file?

Reply:
mmm probably yes... It worked locally but let me check

Reply:
No, it works, only the stdout is returned, stderr if error happens is wrapper in an error instance:
https://github.com/infinispan/infinispan-operator/blob/bdfc2db793ee21e33f89a0d99be491d7c0a95bf6/pkg/kubernetes/kubernetes.go#L150-L159

Reply:
O nice, thanks for double checking :+1: